### DISCUSSION TITLE
Possible open a picture as VJPEG do? Human vision side by side

### DISCUSSION THREAD

**Discussion by donatopace** [d2phap/ImageGlass#1556] [General]
VJPEG tool is light weight only 98K
allows you to open various images on the destop without frames with the right key and each single image can be moved on the screen as if it were a table as well as being able to enlarge them etc.

http://stereopsis.com/vjpeg/
(the same author of PICASA)

Motivation?
The desktop with this software becomes lifelike and allows you to compare images side by side in a very human way.

**Comment by d2phap** [comment]
Hi @donatopace 
FYI, ImageGlass supports frameless window mode. V9.0 also adds support for hotkey customization.

I will not implement many features into ImageGlass to replace other photo viewing apps. It has its own features for different purposes. If you enjoy VJPEG, why don't you just use it together with ImageGlass.

**Reply by donatopace** [reply]
Hi,
Vjpeg only supports images with ICC=sRGB so it's a program that handles Color Management badly.
